packages/toolpad-components/src/Text.tsx

```diff
@@ -241,7 +241,7 @@ function TextContent({ value, loading, sx, variant }: TextContentProps) {
     <MuiTypography
       sx={{
         ...sx,
-        width: '100%',
+        width: 'auto',
```

Comment:
Does it also work if we just omit the width altogether?

Reply:
@apedroferreira yes

Reply:
Ok, if removing it does the job I think it's better to just remove it.